packages/fluentui/ability-attributes/package.json

```diff
@@ -9,7 +9,7 @@
   },
   "devDependencies": {
     "ability-attributes-generator": "^0.0.8",
-    "typescript": "3.7.2"
+    "typescript": "4.1.3"
```

Comment:
It's possible that with the improvements I made to the root `nohoist` config, this won't be necessary now. (The other possibility would be adding a nohoist line for `packages/web-components/**/typescript`.) But might be better to do that separately. ref #14371